README.md

```diff
@@ -1,127 +1,212 @@
-#Breaking Down Methods
+# Writing Methods
 
-Methods in Objective-C may look a little intimidating at first, but all methods can be broken down into five parts:
+## Objectives
 
-1. **Name**
-2. Type: **instance** or **class**
-3. **Arguments** (aka parameters)
-4. **`return`** type 
-5. Associated implementation ("**what does it actually do?**")
+1. Review the steps to calling methods.
+2. Know the syntax for declaring methods.
+3. Recognize when to declare methods as either private or public and how to do so in Objective-C.
+4. Be able to write a method body.
+5. Know where to place `return` in the method body.
 
-Let's use an example to get acquainted with methods.
+## Review: Calling A Method
+
+In the earlier reading about calling methods, we discussed the four elements of making a method call:
+
+```objc
+ReturnType *captureVariable = [recipientObject methodNameArgument:argumentVariable];
+```
+From left to right these are:
+
+1. The variable of the correct type used to capture the return,
+2. The name of the object to perform the method,
+3. The name of the method being called, and
+4. The names of any objects submitted to the methods as arguments. 
+
+**Note:** *Remember, not all methods supply a return or require arguments.*
+
+As you should recall, a **method** is a behavior that an **object** can perform upon itself. Methods are the action sequences of our code. The ability to write custom methods is one of the primary elements of structured programming that makes it a flexible tool for solving a wide variety of problems. Fundamentally, methods allow us to run the exact same section of code multiple times from pretty much anywhere within our application that we wish.
+
+The real beauty of methods lies in not needing to understand all of the ins-and-outs of how any single method works in order to apply that method to given circumstance. Sure, we need to understand *what* the method does in order to apply it appropriately, but that's a far cry from knowing its inner workings. A helpful image is of using a common applicance like a vending machine: you don't need a degree in mechanical engineering in order to trade a handful of coins for a soda—you simply need to meet the input requirements ($1.25 in change) and understand its interface (press a button).
```

Comment:
Thoughts on this insertion, @jmburges ?

Fundamentally, methods allow us to run the exact same section of code multiple times from pretty much anywhere within our application that we wish. On a surface level, methods allow us to repeat an instruction without writing it out twice. This is line with the "Don't Repeat Yourself" (or "DRY") mantra, meaning that any specific instruction should only be written once. 

If we were to copy/paste a line of code even just handful of times, updating it or fixing a bug in it quickly becomes a tedious set of repetitions. And, if the pasted code is dispersed throughout an application, it can be difficult to track down each copy of it down when a change is needed. However, building the instruction into a method which can then be called from elsewhere in you code allows a single update to the instruction to affect *for certain* every case of it running.

The real beauty of methods...